config/default.json

```diff
@@ -2,6 +2,7 @@
   "mode": "bundled",
   "port": 8081,
   "anchorControllerEnabled": false,
+  "schedulerIntervalMS": 300000,
   "cronExpression": "0/5 * * * ? *",
```

Comment:
can we remove all references to `cronExpression` now everywhere in this repo?

Reply:
Removed all references